js/projects/projectsView.js

```diff
@@ -25,6 +47,36 @@ projectsView.handleMainNav = function() {
 
 };
 
-$(document).ready(function(){
-    projectsView.handleMainNav();
+projectsView.handleTruncatedDescription = function() {
+  var minimizedElements = $('.project-description p');
+  console.log(minimizedElements);
+  var minimizeCharacterCount = 100;
+
+  minimizedElements.each(function(){
+    var t = $(this).text();
+    if(t.length < minimizeCharacterCount ) return;
+
+    $(this).html(
+            t.slice(0,minimizeCharacterCount )+'<span>... </span><a href="#" class="more">More</a>'+
+            '<span class="less">'+ t.slice(minimizeCharacterCount ,t.length)+' <a href="#" class="less">Less</a></span>'
+        );
```

Comment:
I noticed a couple of things here, one you are using the less class on two different elements it may cause unexpected issues when you go to style the more/less feature. Also seeing the page the more/less feature need to pop more, it is currently blending in with the description text.